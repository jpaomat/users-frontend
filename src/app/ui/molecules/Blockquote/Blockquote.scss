.blockquote {
    width: 400px;
    min-width: 300px;
    max-width: 500px;
}
