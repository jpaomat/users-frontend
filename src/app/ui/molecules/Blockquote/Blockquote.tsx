import { IBlockquote } from '../../interfaces';

export const Blockquote = ({description, footerText}: IBlockquote) => {
    return (
		<div>
			<blockquote className='blockquote'>
				<p className='font-16'>{description}</p>
			</blockquote>
			<figcaption className='blockquote-footer'>
				<cite title='Source Title'>{footerText}</cite>
			</figcaption>
		</div>
	);
};
