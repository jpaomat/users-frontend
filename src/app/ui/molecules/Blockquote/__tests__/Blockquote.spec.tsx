import { render } from '@testing-library/react';
import { defaultProps } from './mocks';
import { Blockquote } from '../Blockquote';

describe('<BlockquoteUser /> component', () => {

	beforeEach(()=> jest.clearAllMocks);
	
	test('should render component', () => {
		const { container } = render(
				<Blockquote {...defaultProps}/>
		);
		expect(container).toMatchSnapshot();
	});
});
