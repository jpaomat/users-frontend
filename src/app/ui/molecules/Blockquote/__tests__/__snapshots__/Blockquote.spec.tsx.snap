// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockquoteUser /> component should render component 1`] = `
<div>
  <div>
    <blockquote
      class="blockquote"
    >
      <p
        class="font-16"
      >
        text blockquote
      </p>
    </blockquote>
    <figcaption
      class="blockquote-footer"
    >
      <cite
        title="Source Title"
      >
        text
      </cite>
    </figcaption>
  </div>
</div>
`;
